client/mobile/components/student/responses/multiChoice.js

```diff
@@ -55,54 +55,25 @@ class MultiChoice extends React.Component {
 
   renderButton(value) {
     return (
-      <TouchableHighlight underlayColor='#FF0000' style={styles.button} onPress={this.submitAnswer.bind(this, value)}>
-        <Text>{value}</Text>
-      </TouchableHighlight>
+      <Button onPress={this.submitAnswer.bind(this, value)} text={value} />
     )
   }
 
   render() {
     return (
-      <View style={styles.container}>
-        <View style={styles.halfHeight1}>
-          <Text onPress={this.previousPage.bind(this)}>back</Text>
+      <View style={{flex:1}}>
+        <View>
+          <NavBar navi={this.props.navigator} onBack={this.previousPage.bind(this)}>MultiChoice</NavBar>
         </View>
-        <View style={styles.halfHeight}>
-          <Text>MultiChoice</Text>
+        <View style={{flex: 1, justifyContent: 'center', alignItems: 'center' }}>
+          {this.renderButton("A")}
```

Comment:
Do we need to call the render button function, or can we just put in the button?